website/www/check-links.sh

```diff
@@ -73,7 +73,7 @@ for external_link in "${external_links[@]}"
 do
     redraw_progress_bar 50 1 $i ${#external_links[@]}
 
-    if ! curl -sSfL --max-time 10 --connect-timeout 10 --retry 2 -4 "${external_link}" > /dev/null ; then
+    if ! curl -sSfL --max-filesize 2000000 --max-time 10 --connect-timeout 10 --retry 2 -4 "${external_link}" > /dev/null ; then
```

Comment:
What happens for files greater than this size? If the answer is just, they don't get downloaded, can we set this even lower?

Reply:
correct. curl errors out in that scenario, which causes the files to be added to the list of troublemakers - but I think this may be fine. I can add some printed output and comments stating this.

Reply:
done. adjusted to 1mb